CONTRIBUTING.md

```diff
@@ -1,109 +1,113 @@
-## Contribution Guidelines
+# Contribution Guidelines
 
-Crayons is a component library based on Web Components. The goal of Crayons is to provide the developer with easy to use web components by reducing the number of lines of code while giving your app a Freshworks product like user interface. 
+## About Crayons
 
+Crayons is a component library based on Web Components. The goal of Crayons is to provide the developer with easy to use web components by reducing the number of lines of code while giving your app a Freshworks product like user interface.
 
-#### About Crayons
-Crayons are built using StencilJS which provides all features of web component. 
+Crayons are built using StencilJS which provides all features of web component.
 
-#### Crayons Important links 
-- [Repository](https://github.com/freshdesk/crayons) 
+### Crayons Important links
+
+- [Repository](https://github.com/freshdesk/crayons)
 - [Documentation](https://crayons.freshworks.com)
 
-### How to Contribute
-Crayons have a predefined set of components, As a developer, you can not only use the components in your projects, you can also contribute to Crayons as it falls under MIT license and we welcome all contributions, small or big. 
+## How to Contribute
+
+Crayons have a predefined set of components, As a developer, you can not only use the components in your projects, you can also contribute to Crayons as it falls under MIT license and we welcome all contributions, small or big.
 
-*Contributing usually requires one or more of the following steps* 
+### Contributing usually requires one or more of the following steps
 
-1. Forking and setting up 
+1. Forking and setting up
 2. Raise issues towards the repo
 3. Fixing existing or a new issue
 4. Creating a new component
 
-*Setup the local environment*
-1. Fork the crayons repository
-2. Clone the forked repo
-3. Change directory to the cloned repository and install all the dependencies using *npm install*
-4. Run `git config core.hooksPath .git/hooks/`
-5. Run `npm run dev` to open storybook in `https://localhost:9000`
+### Setup the local environment
+
+1. Fork the Crayons repository.
+2. Clone your forked Crayons repository.
+3. Change directory to the cloned repository and install all the dependencies using `npm install`.
+4. Run `git config core.hooksPath .git/hooks/`.
+5. Run `npm run dev` to open storybook in `https://localhost:9000`.
 
 `
 Note : Please use npm instead of yarn
 `
 
-*Raise an issue in Crayon*
+### Raise an issue in Crayon
 
-If you find an issue or is you want to raise a feature request in, make sure you raise an issue in https://github.com/freshdesk/crayons/issues
+If you find an issue or is you want to raise a feature request in, make sure you raise an issue in <https://github.com/freshdesk/crayons/issues>
```

Comment:
This breaks in Github display

Reply:
It looks good on the branch [here](https://github.com/freshdesk/crayons/blob/issue-templates/CONTRIBUTING.md#raise-an-issue-in-crayon). Could you show any screenshot please?

Reply:
OK. It looks fine now. I don't know why it broke the other time.